zebra/zebra_rib.c

```diff
@@ -3139,12 +3139,15 @@ void rib_sweep_table(struct route_table *table)
 	struct route_entry *re;
 	struct route_entry *next;
 	struct nexthop *nexthop;
+	int64_t now = zrouter.startup;
 
 	if (!table)
 		return;
 
 	for (rn = route_top(table); rn; rn = srcdest_route_next(rn)) {
 		RNODE_FOREACH_RE_SAFE (rn, re, next) {
+			int64_t re_time = re->uptime;
```

Comment:
Where are we populating re->uptime, my guess is while adding route to rib. May be this change is part of some recent PR, so just wanted to confirm.

Reply:
all places that we create a re, we immediately set the uptime and has been this way since the code was written.